src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -99,10 +112,6 @@ public function hasListeners($event)
     public function addEventListener($events, $listener)
     {
         if (\is_string($listener)) {
-            if ($this->initialized) {
-                throw new \RuntimeException('Adding lazy-loading listeners after construction is not supported.');
```

Comment:
means this can be removed?

https://github.com/symfony/symfony/blob/43205af51eb46ffb08aa2b82c8b7c76b7020826a/src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php#L110